src/sentry/models/integrations/project_integration.py

```diff
@@ -14,7 +14,7 @@ class ProjectIntegration(Model):
     __relocation_scope__ = RelocationScope.Global
 
     project = FlexibleForeignKey("sentry.Project")
-    integration_id = HybridCloudForeignKey("sentry.Integration", on_delete="CASCADE")
+    integration_id = HybridCloudForeignKey("sentry.Integration", on_delete="DO_NOTHING")
```

Comment:
I needed a model to test the behaviour on and found this deprecated model with  only one record in prod. App behaviour should be unaffected by this since no feature actively uses this model.